<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterlisImportData</class>
 <widget class="QDialog" name="InterlisImportData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Interlis Data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>88</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Target</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="type_combo_box">
       <item>
        <property name="text">
         <string>PostGIS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GeoPackage</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="ili_config">
         <property name="title">
          <string>Interlis</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="4">
           <widget class="QToolButton" name="xtf_file_browse_button">
            <property name="toolTip">
             <string>Browse XTF files</string>
            </property>
            <property name="text">
             <string>…</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Models</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QToolButton" name="multiple_models_button">
            <property name="toolTip">
             <string>Select multiple Interlis models</string>
            </property>
            <property name="text">
             <string>…</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3" colspan="2">
           <widget class="QPushButton" name="ili2db_options_button">
            <property name="text">
             <string>Advanced Options</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="CompletionLineEdit" name="ili_models_line_edit" native="true"/>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QLineEdit" name="xtf_file_line_edit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>XTF File</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>3</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="3" colspan="2">
           <widget class="QCheckBox" name="chk_delete_data">
            <property name="toolTip">
             <string>Delete existing data including dependencies like catalogues etc.</string>
            </property>
            <property name="text">
             <string>Delete existing data in affected tables</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="toml_file_info_label">
            <property name="maximumSize">
             <size>
              <width>580</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="db_wrapper_group_box">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="db_layout"/>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QTextBrowser" name="txtStdout">
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="progress_bar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="import_tool_button">
       <property name="text">
        <string>...</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CompletionLineEdit</class>
   <extends></extends>
   <header>QgisModelBaker.gui.options</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>type_combo_box</tabstop>
  <tabstop>xtf_file_line_edit</tabstop>
  <tabstop>xtf_file_browse_button</tabstop>
  <tabstop>ili_models_line_edit</tabstop>
  <tabstop>multiple_models_button</tabstop>
  <tabstop>chk_delete_data</tabstop>
  <tabstop>txtStdout</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InterlisImportData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InterlisImportData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
